form.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="styles.css" />
+    <script type="text/javascript" src="js/form.js" defer></script>
+  </head>
+  <body data-js="body">
+    <header class="header" data-js="header">
+      <h1>Quiz App</h1>
+    </header>
+    <main>
+        <form data-js="question-form">
+            <fieldset>
```

Comment:
You are currently grouping all of your form input fields. As long as you don't want to group just some fields you do not have to use fieldset.